mongoose.h

```diff
@@ -2961,6 +2963,57 @@ struct mg_profitem {
 #endif
 
 
+#if MG_ENABLE_TCPIP && defined(MG_ENABLE_DRIVER_CYW) && MG_ENABLE_DRIVER_CYW
+
+struct mg_tcpip_spi_ {
+  void *spi;              // Opaque SPI bus descriptor
+  void (*begin)(void *);  // SPI begin: slave select low
+  void (*end)(void *);    // SPI end: slave select high
+  void (*txn)(void *, uint8_t *, uint8_t *,
+              size_t len);  // SPI transaction: write-read len bytes
+};
```

Comment:
How is it different from the mg_tcpip_spi ?

Reply:
The existing function transfers one byte. This one transfers a bunch of bytes in one transaction.
Initially it was intended to transfer 16-bit or 32-bit words, it may get there given enough time.